src/main/resources/cms.conf

```diff
@@ -198,3 +198,4 @@ cms.iam.token.ttl=1h
 cms.user.groups.caseSensitive=true
 
 cms.encryption.cache.enabled=false
+cms.iam.token.cache.enable=false
```

Comment:
Can you rename to `cms.auth.iam.kms.cache.enabled`

Reply:
There's another config `cms.iam.token.ttl` that's somewhat related. But I can change it.